src/scale.ts

```diff
@@ -332,6 +335,11 @@ export function channelScalePropertyIncompatability(channel: Channel, propName:
         return log.message.CANNOT_USE_PADDING_WITH_FACET;
       }
       return undefined; // GOOD!
+    case 'interpolate':
```

Comment:
Not sure why you don't follow my suggestion to combine this with scheme and make the log message take a function.  

It's a minor point, but that way the code seems a bit shorter.